public/csv/bulk_create_trainee.csv

```diff
@@ -0,0 +1 @@
+﻿Provider Trainee ID,Application ID,HESA ID,First Names,Last Name,Previous Last Name,Date of Birth,NI Number,Sex,Email,Nationality,Ethnicity,Disability 1,Disability 2,Disability 3,Disability 4,Disability 5,Disability 6,Disability 7,Disability 8,Disability 9,ITT Aim,Training Route,Qualification Aim,Course Subject One,Course Subject Two,Course Subject Three,Study Mode,ITT Start Date,ITT End Date,Course Age Range,Course Year,Lead Partner URN,Employing School URN,Trainee Start Date,PG Apprenticeship Start Date,First Placement URN,Fund Code,Funding Method,Training Initiative,Additional Training Initiative,UK Degree Type,Non-UK Degree Type,Degree Subject,Degree Grade,Degree Graduation Year,Degree UK Awarding Institution,Degree Country

```

Comment:
I think this differs slightly from the [latest version of the CSV in SharePoint.](https://educationgovuk.sharepoint.com/:x:/r/sites/TeacherServices/Shared%20Documents/Becoming%20a%20Teacher/TWD-REGISTER/Business%20Analyst/Bulk%20Upload%20CSV/Bulk%20Upload%20CSV%20Vendor%20Template/csv/Bulk_Upload_Sheet_V.0.3.csv?d=w4430233c0f9d42af821855e2037d397f&csf=1&web=1&e=roFO34)

```suggestion
﻿Provider Trainee ID,Application ID,HESA ID,First Names,Last Name,Previous Last Name,Date of Birth,NI Number,Sex,Email,Nationality,Ethnicity,Disability 1,Disability 2,Disability 3,Disability 4,Disability 5,Disability 6,Disability 7,Disability 8,Disability 9,ITT Aim,Training Route,Qualification Aim,Course Subject One,Course Subject Two,Course Subject Three,Study Mode,ITT Start Date,ITT End Date,Course Age Range,Course Year,Lead Partner URN_UKPRN,Employing School URN,Trainee Start Date,PG Apprenticeship Start Date,First Placement URN,Fund Code,Funding Method,Training Initiative,Additional Training Initiative,UK Degree Type,Non-UK Degree Type,Degree Subject,Degree Grade,Degree Graduation Year,Degree UK Awarding Institution,Degree Country

```